shared/spotify-api/scripts/generate-mocks.ts

```diff
@@ -135,6 +170,19 @@ async function getTrack(id: string) {
   );
 }
 
+async function getEpisode(id: string) {
+  return tap(
+    (store.episodes[id] ||= await get('/episodes/:id', { id })),
+    async (episode) => {
+      episode.show = await getShow(episode.show.id);
```

Comment:
If possible, I'd like to make the mock data as "pure" as possible, meaning we should avoid overriding fields (after all, the resolvers expect data in the shape of the actual API).

In this case, I'd just remove the `episode.show =` part of this code.

```suggestion
      await getShow(episode.show.id);
```

Reply:
Makes sense! I think I copied this from the equivalent from tracks, so do we want to update that as well?